hack/k8s/helm/nuclio/templates/deployment/dashboard.yaml

```diff
@@ -232,10 +232,12 @@ spec:
         args:
           - "run"
           - "--server"
+          - "--addr=0.0.0.0:8181"
```

Comment:
how about adding port to values and then pass to `containerPort: 8181` and to this string as `{{ .Values.dashboard.opa.port }}`?

Reply:
Done